autohooks/plugins/pylint/pylint.py

```diff
@@ -36,18 +39,18 @@ def check_pylint_installed():
         ) from e
 
 
-def get_pylint_config(config):
+def get_pylint_config(config : AutohooksConfig) :
     return config.get("tool").get("autohooks").get("plugins").get("pylint")
 
 
-def ensure_iterable(value):
+def ensure_iterable(value) -> List[Any]:
```

Comment:
```suggestion
def ensure_iterable(value) -> Iterable:
```

`return value` can be anything ... basically the function `ensure_iterable` is not ensuring iterable for anything but string :D

Reply:
What do you think of my new annotatin? I marked "value" as string or list of strings (like its applied in the code). I believe in that way its a clean solution. Or do you still prefer your proposal? My thought was that the return value doesnt even have to be an Iterable if the parameter is for example an boolean.
`def ensure_iterable(value : Union[str,List[str]]) -> List[str]:`